### DISCUSSION TITLE
Import components from Storybook?

### DISCUSSION THREAD

**Discussion by brenton-klassen-abb** [teleporthq/teleport-code-generators#712] [Q&A] [answered]
Is it possible to import the company's existing React component library (e.g. Storybook) into the platform to enable visually building pages using existing React components?

**Comment by paulbrie** [comment, marked as answer]
Not directly. We an experimental intermediary form that I can demo to you if you're interested.

**Reply by brenton-klassen-abb** [reply]
Is it in the docs? I searched the site but couldn't find an answer to this question of importing existing components.
